### ISSUE TITLE
Deal with maui__G20, pgf1__G08, pgf5__G20 in the validation set

### ISSUE THREAD

**Issue by vc1492a** [vc1492a/tidd#82]
Many of the satellites (need to check all again not just those in the title of the issue) do not have validation data (images) on the day of the earthquake. In the interest of speed, it may be best to simply replace those with full-formed examples from the training set (this will reduce the training set slightly but I do not expect this to be a performance issue).
